src/modules/mainSec.js

```diff
@@ -1,72 +1,99 @@
 import NewItem from './listClass.js';
 import * as selectors from './selectors.js';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Linters help us to write clean code so disabling them means we are doing ourselves a disservice.

- [ ] Please kindly enable them and fix them instead.